lib/mapper.js

```diff
@@ -164,14 +168,16 @@ function createBaseTrackProperties(track, settings) {
   if (settings.prefixProperties) {
     properties = prefix(track.event(), properties);
   }
-
-  return extend(
-    properties,
-    {
+  // Only apply userAgent property on `visited site` event
+  if (track.event() === 'visited site') {
+    properties._userAgent = userAgent || track.proxy('context.userAgent');
+  }
```

Comment:
This implementation will require changes to `data-translator`. Currently we only parse the actual `User-Agent` header in the raw request. This is adding it as a separate property, which is not currently handled. @paulgoleary